website/docs/advanced/override_grammar/basic.md

```diff
@@ -147,6 +147,19 @@ Quoted strings can accept any value between the quotes, but some characters need
 </div>
 </div>
 
+It may be necessary to use multiple pairs of quotes to prevent your
+shell from consuming quotation marks before they are passed to hydra.
+
+```shell
+$ python my_app.py +foo="{a: 10}"  # the string `+foo={a: 10}` is passed to Hydra by bash
+foo:
+  a: 10
```

Comment:
I think this might be confusing. since we recommend quote key=value pare in single quote, maybe update this to 
```
python my_app.py '+foo={a: 10}'
```

Reply:
Updated in https://github.com/facebookresearch/hydra/pull/1844/commits/d30001657e95418436b6bb234d02131c2b29b2ac.